modules/Bio/EnsEMBL/Compara/PipeConfig/UpdateReferenceDatabase_conf.pm

```diff
@@ -0,0 +1,307 @@
+=head1 LICENSE
+
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::PipeConfig::UpdateReferenceDatabase_conf
+
+=head1 DESCRIPTION
+
+    This is a PipeConfig for TODO
+
+=cut
+
+package Bio::EnsEMBL::Compara::PipeConfig::UpdateReferenceDatabase_conf;
+
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Hive::Version 2.4;
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+
+use Bio::EnsEMBL::Compara::PipeConfig::Parts::DumpFastaDatabases;
+
+use base ('Bio::EnsEMBL::Compara::PipeConfig::ComparaGeneric_conf');
+
+sub no_compara_schema {};
+
+sub default_options {
+    my ($self) = @_;
+
+    return {
+        %{$self->SUPER::default_options},   # inherit the generic ones
+
+        'division'     => 'references',
+        'ref_db'       => 'compara_references',
+        'taxonomy_db'  => 'ncbi_taxonomy',
+
+        # how many parts should per-genome files be split into?
+        'num_fasta_parts'  => 100,
+        # at which id should genome_db start?
+        'genome_db_offset' => 101,
```

Comment:
I don't think we want to avoid a hard limit of the number of query species. This is supposed to be for a "rapid release" and one of the best ways to keep it rapid, is to keep it small. The hard limit is not limited to the sole reason of avoiding overlapping ids. Another very basic reason is the human readability, query and reference can be easily visibly distinguishable in the homology annotation pipeline in this way.